### PR TITLE
Add dashboard and events page

### PR DESCRIPTION
**Pull request by nelsonmestevao**
**Events page**

image

**Event popup**

image

**Dashboard**

image

### CODE REVIEW

**Comment by nelsonmestevao** [comment]
#### 📝Changed routes:


| [**/dashboard**](https://shuriken-e4yc9sgcx-cesiuminho.vercel.app/dashboard) | [**/**](https://shuriken-e4yc9sgcx-cesiuminho.vercel.app/) |
|:-:|:-:|
|<a href="https://shuriken-e4yc9sgcx-cesiuminho.vercel.app/dashboard">Screenshot of /dashboard</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/woV52zvf3l93qAuuuagzkTMxXjn0xvINTX4YVUPjTinmGbKeB1YCn6EJRm4F4kzjpTZ3dDrbgWGZvKjOW6HuiA.png">(view full size)</a> |<a href="https://shuriken-e4yc9sgcx-cesiuminho.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/blL4U-pa5kMh1jJCqqo5AJH9H_GLxXEs4vvlVEFw_zHtS1FpuWIKdD32ka8tzRomWkz3T3_GGO0sOMQ-j8rkiQ.png">(view full size)</a> |







Commit f1b239dd3116bab4e65d506650cf93ebb634c53c (https://shuriken-e4yc9sgcx-cesiuminho.vercel.app).

File: components/Event/EventInfo.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { DateTime } from "luxon";
```
Comment (miguelbrandao) [inline components/Event/EventInfo.js:1]: Is this dependency really necessary? Isn't [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) enough?

**Review by miguelbrandao — APPROVED** [review]
It looks good! I know it's a WIP, but if you wanna merge it as it is go for it.

File: components/Event/EventInfo.js (hunk 1 above)
Comment (nelsonmestevao) [reply components/Event/EventInfo.js:1]: I guess for now it is :disappointed:

**Pull request merged**